### PR TITLE
Pinterest advertiser addition

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

This PR adds the `advertiser_id` and `advertiser_name` to the `stg_pinterest_ads` staging model to be rolled downstream into the `ad_reporting` model as the account_id and account_name fields.

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide explanation how the change is non breaking below.)
This will only add the advertiser information for Pinterest users as the account information. This will not result in a breaking change.

**Is this PR in response to a previously created Issue**

- [X] Yes, Issue [#16](https://github.com/fivetran/dbt_pinterest/issues/16)
- [ ] No 

**How did you test the PR changes?** 

 
- [X] CircleCi  
- [ ] Other (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 
- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [X] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🔢 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/stg_pinterest_ads.sql [hunk 1]
```diff
@@ -9,6 +9,8 @@ with base as (
 
     select
         cast(campaign_date as date) as date_day,
+        cast(advertiser_id as {{ dbt_utils.type_string() }}) as accound_id,
```
Comment (fivetran-sheringuyen) [inline models/stg_pinterest_ads.sql:12]: *Account_id

**Review by fivetran-sheringuyen — COMMENTED** [review]
ran `dbt run` from local `dev` directory -- these errors probably aren't relevant to these exact changes but curious if you saw them on your end:
`21:20:18  3 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__ad_history_tmp [RUN]
21:20:18  3 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__ad_history_tmp [ERROR in 0.37s]
21:20:18  4 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__ad_set_history_tmp [RUN]
21:20:19  4 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__ad_set_history_tmp [ERROR in 0.42s]
21:20:19  5 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__basic_ad_tmp [RUN]
21:20:19  5 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__basic_ad_tmp [ERROR in 0.34s]
21:20:19  6 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__campaign_history_tmp [RUN]
21:20:19  6 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__campaign_history_tmp [ERROR in 0.41s]
21:20:19  7 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__creative_history_tmp [RUN]
21:20:20  7 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__creative_history_tmp [ERROR in 0.32s]`

`21:23:21  Completed with 5 errors and 0 warnings:
21:23:21  
21:23:21  Runtime Error in model stg_facebook_ads__ad_history_tmp (models/tmp/stg_facebook_ads__ad_history_tmp.sql)
21:23:21    404 Not found: Table dbt-package-testing:facebook_ads.ad_history was not found in location US
21:23:21    
21:23:21    (job ID: 60f637a5-513a-43c5-a8dd-85e91d86cd52)
21:23:21  
21:23:21  Runtime Error in model stg_facebook_ads__ad_set_history_tmp (models/tmp/stg_facebook_ads__ad_set_history_tmp.sql)
21:23:21    404 Not found: Table dbt-package-testing:facebook_ads.ad_set_history was not found in location US
21:23:21    
21:23:21    (job ID: eac868bf-5033-4e31-95a8-509fd66e0e8d)
21:23:21  
21:23:21  Runtime Error in model stg_facebook_ads__basic_ad_tmp (models/tmp/stg_facebook_ads__basic_ad_tmp.sql)
21:23:21    404 Not found: Table dbt-package-testing:facebook_ads.basic_ad was not found in location US
21:23:21    
21:23:21    (job ID: 1fc7022a-4ca4-40b6-8e62-97481011c0f7)
21:23:21  
21:23:21  Runtime Error in model stg_facebook_ads__campaign_history_tmp (models/tmp/stg_facebook_ads__campaign_history_tmp.sql)
21:23:21    404 Not found: Table dbt-package-testing:facebook_ads.campaign_history was not found in location US
21:23:21    
21:23:21    (job ID: 26fc7678-fb99-40ac-8efe-548425f9a1d6)
21:23:21  
21:23:21  Runtime Error in model stg_facebook_ads__creative_history_tmp (models/tmp/stg_facebook_ads__creative_history_tmp.sql)
21:23:21    404 Not found: Table dbt-package-testing:facebook_ads.creative_history was not found in location US
21:23:21    
21:23:21    (job ID: b17b4c9c-a7f2-47ed-a9ab-e831bfda7fb2)
21:23:21  
21:23:21  Done. PASS=89 WARN=0 ERROR=5 SKIP=18 TOTAL=112`

File: models/stg_pinterest_ads.sql (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply models/stg_pinterest_ads.sql:12]: Woah... cant't believe I missed that one. Thanks so much!

**Comment by fivetran-joemarkiewicz** [comment]
> ran `dbt run` from local `dev` directory -- these errors probably aren't relevant to these exact changes but curious if you saw them on your end: `21:20:18 3 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__ad_history_tmp [RUN] 21:20:18 3 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__ad_history_tmp [ERROR in 0.37s] 21:20:18 4 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__ad_set_history_tmp [RUN] 21:20:19 4 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__ad_set_history_tmp [ERROR in 0.42s] 21:20:19 5 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__basic_ad_tmp [RUN] 21:20:19 5 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__basic_ad_tmp [ERROR in 0.34s] 21:20:19 6 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__campaign_history_tmp [RUN] 21:20:19 6 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__campaign_history_tmp [ERROR in 0.41s] 21:20:19 7 of 112 START view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__creative_history_tmp [RUN] 21:20:20 7 of 112 ERROR creating view model dbt_sheri_stg_facebook_ads.stg_facebook_ads__creative_history_tmp [ERROR in 0.32s]`
> 
> `21:23:21 Completed with 5 errors and 0 warnings: 21:23:21 21:23:21 Runtime Error in model stg_facebook_ads__ad_history_tmp (models/tmp/stg_facebook_ads__ad_history_tmp.sql) 21:23:21 404 Not found: Table dbt-package-testing:facebook_ads.ad_history was not found in location US 21:23:21 21:23:21 (job ID: 60f637a5-513a-43c5-a8dd-85e91d86cd52) 21:23:21 21:23:21 Runtime Error in model stg_facebook_ads__ad_set_history_tmp (models/tmp/stg_facebook_ads__ad_set_history_tmp.sql) 21:23:21 404 Not found: Table dbt-package-testing:facebook_ads.ad_set_history was not found in location US 21:23:21 21:23:21 (job ID: eac868bf-5033-4e31-95a8-509fd66e0e8d) 21:23:21 21:23:21 Runtime Error in model stg_facebook_ads__basic_ad_tmp (models/tmp/stg_facebook_ads__basic_ad_tmp.sql) 21:23:21 404 Not found: Table dbt-package-testing:facebook_ads.basic_ad was not found in location US 21:23:21 21:23:21 (job ID: 1fc7022a-4ca4-40b6-8e62-97481011c0f7) 21:23:21 21:23:21 Runtime Error in model stg_facebook_ads__campaign_history_tmp (models/tmp/stg_facebook_ads__campaign_history_tmp.sql) 21:23:21 404 Not found: Table dbt-package-testing:facebook_ads.campaign_history was not found in location US 21:23:21 21:23:21 (job ID: 26fc7678-fb99-40ac-8efe-548425f9a1d6) 21:23:21 21:23:21 Runtime Error in model stg_facebook_ads__creative_history_tmp (models/tmp/stg_facebook_ads__creative_history_tmp.sql) 21:23:21 404 Not found: Table dbt-package-testing:facebook_ads.creative_history was not found in location US 21:23:21 21:23:21 (job ID: b17b4c9c-a7f2-47ed-a9ab-e831bfda7fb2) 21:23:21 21:23:21 Done. PASS=89 WARN=0 ERROR=5 SKIP=18 TOTAL=112`

What were the contents of your `dbt_project.yml`? It looks like there was a mis-config with facebook ads

File: models/stg_pinterest_ads.sql (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply models/stg_pinterest_ads.sql:12]:
```suggestion
        cast(advertiser_id as {{ dbt_utils.type_string() }}) as account_id,
```

**Comment by fivetran-sheringuyen** [comment]
> 

I just used the `dbt init <project>` dbt_project.yml template - with:
`profile: 'fivetran'`

**Comment by fivetran-joemarkiewicz** [comment]
Ah yeah the facebook schema used by default is `facebook_ads`, but we actually would want to override the `facebook_ads_schema` variable to be `facebook_ads_dbt` since that is where the complete set of our facebook data lives.

**Review by fivetran-sheringuyen — APPROVED** [review]
lgtm! no more errors upon fb schema override!

**Comment by fivetran-joemarkiewicz** [comment]
Before merging this I want to 100% confirm that Advertisers share the same functionality as Accounts in the other ad platforms. With the data we have tested this _seems_ to be the case. However, I want to confirm this is before we move forward with this PR.

**Comment by fivetran-joemarkiewicz** [comment]
The updates from this PR are now available in the latest release (`v1.0.0`)! This PR is no longer needed.

**Pull request closed without merging**